spring-cloud-gcp-logging/src/main/java/com/google/cloud/spring/logging/StackdriverJsonLayout.java

```diff
@@ -212,8 +224,11 @@ public void start() {
 					TimeUnit.MILLISECONDS.toNanos(event.getTimeStamp() % 1_000));
 		}
 
-		add(StackdriverTraceConstants.SEVERITY_ATTRIBUTE, this.includeLevel,
-				String.valueOf(event.getLevel()), map);
+		add(StackdriverTraceConstants.SEVERITY_ATTRIBUTE,
+				this.includeLevel,
+				logbackToSeverityMap.getOrDefault(event.getLevel(), Severity.DEFAULT.name()),
```

Comment:
Would you mind adding a small test to exercise the `orDefault` part?

Reply:
Done.